### DISCUSSION TITLE
Läuft Jarvis v3.1.8 auch mit js-Controller v5.0.19?

### DISCUSSION THREAD

**Discussion by mcuiobroker** [Zefau/ioBroker.jarvis#2612] [General]
Gibt es in v3.1.8 Einschränkungen, so dass es unter v5.0.19 nicht läuft? 
Vorab-Frage
Auch in Kombination mit nodejs20?
https://forum.iobroker.net/topic/72949/jarvis-expert-mode-funktioniert-nicht

**Comment by ma-gu-16** [comment]
Also ich hab heute auf 5.0.19 aktualisiert und keine Probleme mit Jarvis. (v3.1.9-beta.2)

Node.js:
v18.19.0
